modules/installation-launching-installer.adoc

```diff
@@ -429,6 +429,14 @@ ifdef::vsphere[]
 .. Specify the user name and password for the vCenter account that has the required permissions to create the cluster.
 +
 The installation program connects to your vCenter instance.
++
+[IMPORTANT]
+====
+Some VMware vCenter Single Sign-On (SSO) environments with Active Directory (AD) integration might primarily require you to use the legacy login method; that is, the `<domain>\` construct.
```

Comment:
Consider:

```suggestion
Some VMware vCenter Single Sign-On (SSO) environments with Active Directory (AD) integration might primarily require you to use the previous login method; that is, the `<domain>\` construct.
```

As per IBMSG: https://www.ibm.com/docs/en/ibm-style?topic=word-usage#legacy

Reply:
I went with "traditional" as previous sounds like othe ld method has been replaced, which is untrue.

Good point. I went with `traditional login method, which requires the `<domain>\` construct.`